avm/res/app/container-app/tests/e2e/max/main.test.bicep

```diff
@@ -41,6 +41,8 @@ module nestedDependencies 'dependencies.bicep' = {
   params: {
     location: resourceLocation
     managedEnvironmentName: 'dep-${namePrefix}-menv-${serviceShort}'
+    keyVaultName: 'dep-${namePrefix}-kv-${serviceShort}'
+    keyVaultSecretName: 'dep-${namePrefix}-kv-secret-${serviceShort}'
```

Comment:
```suggestion
```

As the secret won't conflict with anything, it's good enough to just set a hardcoded value in the deployment itself :) Will suggest an update in the `dependencies.bicep` for it to show what I mean